src/families/cosmos/bridge/libcore.js

```diff
@@ -147,6 +148,14 @@ const getSendTransactionStatus = async (a, t) => {
     errors.amount = new NotEnoughBalance();
   }
 
+  if (
+    a.cosmosResources &&
+    a.cosmosResources.delegations.length > 0 &&
+    t.useAllAmount
+  ) {
+    warnings.emptyBalance = new CosmosEmptyBalanceWithDelegationWarning();
```

Comment:
I agree with this to be fixed, but i'm not sure about the proposal implementation

- emptyBalance is **NOT** an existing fields, warnings and errors are supposed to be attached to field. 

- in Tezos, we do this:

```
        warnings.amount = new RecommendUndelegation();
```

can you please assess if "RecommendUndelegation" wouldn't be convenient to reuse? (prefer reuse of existing errors)
can we please make sure